### PR TITLE
Code coverage for Tomatoes

### PR DESCRIPTION
**Pull request by dalpo**

### CODE REVIEW

File: .simplecov [hunk 1]
```diff
@@ -0,0 +1,4 @@
+SimpleCov.start 'rails' do
+  minimum_coverage(91.65)
```
Comment (potomak) [inline .simplecov]: Is this the current coverage?

Comment (dalpo) [inline .simplecov]: Yes, this is the current coverage by simplecov

**Comment by dalpo** [comment]
@potomak In theory Travis should automatically fail the test pipeline if the coverage goes under the configured limit.

**Comment by dalpo** [comment]
We could also add the code climate coverage badge :)

**Comment by potomak** [comment]
> In theory Travis should automatically fail the test pipeline if the coverage goes under the configured limit.

@dalpo don't we need to add a new item in the `script` list?

I don't see any new info in the PR build: https://travis-ci.org/tomatoes-app/tomatoes/builds/196311786, do you?

**Pull request merged**